models/_sector/referral/rewards/referral_staging_rewards.sql

```diff
@@ -0,0 +1,54 @@
+{{ config(
+    schema = 'referral',
+    alias = alias('staging_rewards'),
```

Comment:
i assume this is the "chain union" step we agreed on.

any reason you went with this naming standard? rather than `referral_<blockchain>.rewards`?

is it because this is a smaller sector that you don't expect to grow much? or should we standardize this to the above like nft/dex?

Reply:
I indeed skipped the chain specific unions here because the sector is small. 
The staging model will also be necessary in nft/dex when we want to split out the enrichment as a view. We'll have a final materialized union (staging or whatever we want to call it) and a final view with the enrichment macro (which is exposed to dune)